index.html

```diff
@@ -288,6 +288,50 @@ <h3 class="skill-name">Skills</h3>
         </ul>
       </section>
       <!-- ./end of about myself section -->
+      <!-- contact form section -->
+      <section class="contact-section">
+        <div class="contact-wrapper">
+          <h1>Contact me</h1>
+          <p>
+            If you have an application you are interested in developing, a
+            feature that you need built or a project that needs coding. I’d love
+            to help with it!
+          </p>
+          <form action="https://formspree.io/f/xdovrpgk" method="POST">
+            <input
+              type="text"
+              name="name"
+              id="name"
+              maxlength="30"
+              placeholder="Lorem Ipsum"
```

Comment:
- [ ] [OPTIONAL] Nice work so far 👍🏿 on using a placeholder for your input but I kindly suggest you use a descriptive placeholder e.g `Enter your full name` to make your project look professional